content/en/docs/zero-code/obi/configure/service-discovery.md

```diff
@@ -0,0 +1,364 @@
+the Pods owned by a `Deployment`, `ReplicaSet`, `DaemonSet`, or `StatefulSet`
+with a name matching the provided glob.
+
+If you specify other selectors in the same `instrument` entry, the processes
+must match all the selector properties.
+
+### K8s Pod labels
+
+This selector property limits the instrumentation to the applications running in
+the Pods with labels matching the provided value as glob.
+
+If you specify other selectors in the same `instrument` entry, the processes
+must match all the selector properties.
+
+For example:
+
+```yaml
+discovery:
+  instrument:
+    - k8s_namespace: frontend
+      k8s_pod_labels:
+        instrument: beyla
+```
+
+The preceding example discovers all Pods in the `frontend` namespace that have a
+label `instrument` with a value that matches the glob `beyla`.
+
+### K8s Pod annotations
+
+This selector property limits the instrumentation to the applications running in
+the Pods with annotations matching the provided value as glob.
+
+If you specify other selectors in the same `instrument` entry, the processes
+must match all the selector properties.
+
+For example:
+
+```yaml
+discovery:
+  instrument:
+    - k8s_namespace: backend
+      k8s_pod_annotations:
+        beyla.instrument: 'true'
+```
+
+The preceding example discovers all Pods in the `backend` namespace that have an
+annotation `beyla.instrument` with a value that matches the glob `true`.
+
+## Survey mode
+
+In survey mode, OBI only performs service discovery and detects the programming
+language of each service, but doesn't instrument any discovered services.
+
+OBI writes the discovered information from survey mode to a metric called
+`survey_info`, which uses the same attributes as the `target_info` metric. The
+Prometheus exporter creates this metric based on the OpenTelemetry metric
+resource attributes. You can use survey mode to build external automated
+instrumentation solutions. For example, you can use the `survey_info` metric to
+list available instrumentation targets and choose which ones to instrument.
+
+Configure the `survey` section exactly like the `instrument` section. For more
+details, see the [discovery services section](#discovery-services) of this
+document.
+
+## Exclude services from instrumentation
+
+The `exclude_instrument` section lets you specify selection criteria for
+excluding services from being instrumented. It follows the same definition
+format as described in the [discovery services](#discovery-services) section of
+this document.
+
+This option helps you avoid instrumenting services typically found in
+observability environments. For example, use this option to exclude
+instrumenting Prometheus.
+
+## Default exclude services from instrumentation
+
+The `default_exclude_instrument` section disables instrumentation of OBI itself
+(self-instrumentation), as well as Grafana Alloy and the OpenTelemetry
+Collector. It also disables instrumentation of various Kubernetes system
```

Comment:
```suggestion
The `default_exclude_instrument` section disables instrumentation of OBI itself
(self-instrumentation), as well the OpenTelemetry Collector and other observability components. 
It also disables instrumentation of various Kubernetes system
```

Reply:
> Ok! (I don't have rights to accept your suggestion)

Another todo, we need to give @open-telemetry/ebpf-instrumentation-approvers permissions on this repo

Reply:
working on that.